cookbooks/S20RTS.prm

```diff
@@ -15,9 +15,12 @@ set Output directory                       = output-S20RTS
 
 # The following variables describe how the pressure should
 # be normalized. Here, we choose a zero average pressure
-# at the surface of the domain 
+# at the surface of the domain. The surface pressure and
+# Adiabatic surface temperature are also used to compute
```

Comment:
```suggestion
# adiabatic surface temperature are also used to compute
```

Reply:
Sorry, missed one thing: Consistent capitalization of `Adiabatic`, or capitalize `Surface pressure` if you intended to match parameter syntax.